### PR TITLE
tpm: Mark Pluton TPMs as part of the main CPU

### PR DESCRIPTION
**Pull request by superm1**
Pluton is on die.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/tpm/fu-tpm-v2-device.c [hunk 1]
```diff
@@ -343,6 +343,10 @@ fu_tpm_v2_device_setup(FuDevice *device, GError **error)
 	tmp = fu_tpm_v2_device_convert_manufacturer(manufacturer);
 	fu_device_set_vendor(device, tmp != NULL ? tmp : manufacturer);
 
+	/* Pluton */
+	if (g_strcmp0(manufacturer, "MSFT") == 0 && g_strcmp0(model, "Pluton.TPM.A") == 0)
+		fu_device_add_internal_flag(device, FU_DEVICE_INTERNAL_FLAG_HOST_CPU_CHILD);
```
Comment (hughsie) [inline plugins/tpm/fu-tpm-v2-device.c:348]: could we do this in a quick file?

Comment (superm1) [reply plugins/tpm/fu-tpm-v2-device.c:348]: How?

Comment (superm1) [reply plugins/tpm/fu-tpm-v2-device.c:348]: I guess maybe matching a guid, I'll look and see if one makes sense.

Comment (hughsie) [reply plugins/tpm/fu-tpm-v2-device.c:348]: Can do something like:

    [TPM\VEN_MSFT&MOD_Pluton.TPM.A&VER_2.0]
    Flags = host-cpu-child

perhaps?

**Review by hughsie — APPROVED** [review]

**Pull request merged**